app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddPitchSurface.kt

```diff
@@ -0,0 +1,56 @@
+package de.westnordost.streetcomplete.quests.surface
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.meta.ALL_ROADS
+import de.westnordost.streetcomplete.data.meta.updateWithCheckDate
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+
+
+class AddPitchSurface : OsmFilterQuestType<SurfaceAnswer>() {
+    private val sportValuesWherePitchSurfaceQuestionIsInteresting = listOf(
+        "soccer", "tennis", "basketball", "equestrian", "athletics", "volleyball", "skateboard",
+        "bmx", "american_football", "rugby_union", "badminton", "pelota", "horse_racing",
+        "disc_golf", "futsal", "cycling", "gymnastics"
+    )
```

Comment:
\+
multi
fitness
bowls, boules
netball
handball, team_handball
hockey, field_hockey
futsal
badminton
padel
pelota
rugby_league
gaelic_games
horseshoes
australian_football
racquet
?

Well, I could go on. Just on page 2 of taginfo.

Reply:
multi - may have "bad" values and will be anyway retagged by higher priority quest for selecting sport of a pitch
fitness - not tagged on pitches, and what is tagged looks in almost all cases like a tagging mistake (leisure=fitness_station)
bowls - on review it makes sense, added
boules - I thought that it is `surface=compacted` but on recheck of what is mapped turned out to be not true, added
netball - rechecked, added
handball, team_handball - makes sense, added
hockey - retagged by sport quest, ambiguous, may mean ice hockey ( https://wiki.openstreetmap.org/wiki/Tag:sport%3Dhockey )
field_hockey - added
futsal - is listed already
badminton - is listed already
padel - added
pelota - is listed already
rugby_league - added
gaelic_games - added
horseshoes - added
australian_football - Wikipedia claims that it is always played on grass, but looking at what is actually tagged it turned to be untrue, added
racquet - added

I must say that blacklist seems to now a better idea than initially.

Reply:
> multi - may have "bad" values and will be anyway retagged by higher priority quest for selecting sport of a pitch 

Not it won't.

Reply:
> Not it won't.

Right, I was convinced that I remember well and failed to verify it. Added to list of accepted values.

Reply:
If anyone sees anything missing/wrong or thinks that further research is needed - let me know.